sound/soc/intel/boards/sof_sdw_max98373.c

```diff
@@ -34,21 +24,19 @@ static int spk_init(struct snd_soc_pcm_runtime *rtd)
 	if (!card->components)
 		return -ENOMEM;
 
-	ret = snd_soc_add_card_controls(card, mx8373_controls,
-					ARRAY_SIZE(mx8373_controls));
+	ret = snd_soc_add_card_controls(card, maxim_kcontrols, 2);
```

Comment:
Why don't we use `ARRAY_SIZE(maxim_kcontrols)`? Isn't `ARRAY_SIZE(maxim_kcontrols)` = 2?